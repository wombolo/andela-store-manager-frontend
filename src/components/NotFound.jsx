import React, { Component } from 'react';

const NotFound = ({ location }) => (
  <div>
    <h3>Specified route <code>{location.pathname}</code> does not exist</h3>
  </div>
);

export default NotFound;
