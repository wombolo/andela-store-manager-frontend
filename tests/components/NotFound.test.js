import 'babel-polyfill'
import React from 'react';
import { shallow } from 'enzyme';

import NotFound  from '../../src/components/NotFound.jsx';

let wrapper;

const props = {
  location:{pathname: '/not-found-link'}
};

describe('NotFound Component', () => {
  it('should match snapshot', () => {
    wrapper = shallow(<NotFound {...props}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
