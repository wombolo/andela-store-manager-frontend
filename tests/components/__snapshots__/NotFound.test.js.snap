// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component should match snapshot 1`] = `
<div>
  <h3>
    Specified route 
    <code>
      /not-found-link
    </code>
     does not exist
  </h3>
</div>
`;
